### PR TITLE
Added attribute rel="prev/next" for SEO

### PR DESCRIPTION
**Pull request by geekdevs**
Added attribute rel="prev/next" for SEO
This is followup for PR https://github.com/whiteoctober/Pagerfanta/pull/132
Rebased from master and solved conflicts.

### CODE REVIEW

File: src/Pagerfanta/View/Template/DefaultTemplate.php [hunk 1]
```diff
@@ -24,8 +24,10 @@ class DefaultTemplate extends Template
         'css_current_class'  => 'current',
         'dots_text'          => '...',
         'container_template' => '<nav>%pages%</nav>',
-        'page_template'      => '<a href="%href%">%text%</a>',
+        'page_template'      => '<a href="%href%"%rel>%text%</a>',
```
Comment (stof) [inline src/Pagerfanta/View/Template/DefaultTemplate.php]: should be ``%rel%``. Existing placeholders are enclosed in ``%``

File: src/Pagerfanta/View/Template/DefaultTemplate.php [hunk 2]
```diff
@@ -40,12 +42,12 @@ public function page($page)
         return $this->pageWithText($page, $text);
     }
 
-    public function pageWithText($page, $text)
+    public function pageWithText($page, $text, $rel = null)
     {
-        $search = array('%href%', '%text%');
+        $search = array('%href%', '%text%', '%rel');
```
Comment (stof) [inline src/Pagerfanta/View/Template/DefaultTemplate.php]: should be ``%rel%``

**Comment by geekdevs** [comment]
@stof updated placeholder naming

**Comment by geekdevs** [comment]
ping?

**Comment by odolbeau** [comment]
Any news on this PR?

**Comment by richsage** [comment]
Happy with this - thanks! :-)

**Pull request merged**